### PR TITLE
js without jQuery

### PR DESCRIPTION
**Pull request by taisukef**
simplified js code as same functions without jQuery

### CODE REVIEW

File: websocket-chat/static/websocket.html [hunk 1]
```diff
@@ -1,90 +1,79 @@
...
-      $('#text').keyup(function(e) {
-        if (e.keyCode === 13) {
-          $('#send').click();
-          return false;
-        }
-      });
-    });
+<!DOCTYPE html><html><meta charset="utf-8"><head>
+<title>Chat!</title>
+<script type="module">
```
Comment (JohnTitor) [inline websocket-chat/static/websocket.html:3]: I think this type declaration is unnecessary?

Comment (taisukef) [reply websocket-chat/static/websocket.html:3]: Thank you for your review.
if declare type="module", the browser parse as the strict mode for solid program, and the script can use the ES module import.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode

Comment (robjtede) [reply websocket-chat/static/websocket.html:3]: The script doesn't use imports and would improve browser compatibility slightly. A 'use strict' declaration would do the same job.

File: websocket-chat/static/websocket.html [hunk 2]
```diff
@@ -1,90 +1,79 @@
...
+window.onload = () => {
+  let conn = null;
+  const log = msg => {
+    div_log.innerHTML += msg + '<br>';
+    div_log.scroll(0, div_log.scrollTop + 1000);
+  };
+  const connect = () => {
+    disconnect();
+    const wsUri = (window.location.protocol === 'https:' && 'wss://' || 'ws://') + window.location.host + '/ws/';
```
Comment (robjtede) [inline websocket-chat/static/websocket.html:12]:
```suggestion
    const wsUri = (window.location.protocol === 'https:' ? 'wss://' : 'ws://') + window.location.host + '/ws/';
```

**Comment by robjtede** [comment]
Thanks for this move off jQuery. Can you run the HTML file through prettier or some similar formatter, too?

**Comment by robjtede** [comment]
hey @taisukef, can you rebase on master and double check this works with v3

**Review by robjtede — APPROVED** [review]

**Pull request merged**